docs/learn/parimutuel.md

```diff
@@ -0,0 +1,151 @@
+---
+id: parimutuel
+title: Parimutuel
+---
+
+## Overview
+
+The term _parimutuel_ refers to a particular market making and payout mechanism
+used on Zeitgeist for extra casual markets.
+
+These are "losers pay winners" market makers: Any informant can bet any amount
+at any time. Their bet amount goes into the _pot_ and they receive tokens which
+represent their share of the pot. After the market is resolved, the entire pot
+is distributed amongst those who wagered on the outcome that materialized,
+proportional to what their share of the pot is.
+
+Although parimutuels work with scalar markets, Zeitgeist currently only supports
+parimutuel for categorical markets.
```

Comment:
I don't think we need to explain why we don't implement certain features. I also disagree with some of what you write. I don't think there's any problem with scalar markets in conjunction with parimutuels that you don't also have with categorical markets.

Reply:
> I don't think we need to explain why we don't implement certain features.

Ok.

> I don't think there's any problem with scalar markets in conjunction with parimutuels that you don't also have with categorical markets.

Please think the reward process through with scalar markets. I thought we came to the same conclusion in our process and for that reason we decided to remove scalar markets for Parimutuels. Didn't we?

For categorical markets, if you buy the winning outcome, you can not lose any base asset. For scalar markets you can lose base asset, because there is no clear winner of LONG or SHORT, because the pot is divided by the actual scalar value between range start and range end.

Reply:
There's a clear winning strategy for scalar markets if your prediction is correct: If you think that the long correctness is c, and thus the short correctness is (1-c), then buy xc units of LONG and x(1-c) units of SHORT. The payoff is x(cr_l + (1 - c)r_s). You can know show that this greater or equal to x and equality holds if and only if everybody else is also buying at the same ratio.

The approach to do this is to define y = a_s/a_l and then calculate

cr_l + (1-c)r_s = c^2 * (1 + y) + (1 - c)^2 * (1 + 1/y).

You can now define f(c, y) = c^2 * (1 + y) + (1 - c)^2 * (1 + 1/y) and take first and second partial derivatives to see that in the strip (0, 1) \times (0, \infty), we have local minima at c = 1/(y + 1), and f(1/(y + 1), y) = 1, which proves the assertion (takes a bit of calculation to get through this).

TL;DR: Betting on parimutuel scalar markets just works a bit different than with AMM or CDA. You don't bet based on market state because... well, there is no market state.

Reply:
The doctor himself spoke.